app/templates/components/tabbed-navigation.hbs

```diff
@@ -1,8 +1,8 @@
 {{#if device.isMobile}}
-  <div role="button" class="ui segment center aligned" {{action 'toggleMenu'}}>
+  <div role="button" class="ui segment center aligned" onClick={{action 'toggleMenu'}}>
```

Comment:
@niranjan94 I understand that an action by default binds to the click event but if I remove onClick then the value of the is not getting updated though the problem of misalignment is resolved. If we navigate then the value remains same that is why I have explicitly called `onClick` here.